storage_service/locations/models/wellcome.py

```diff
@@ -69,14 +69,42 @@ def browse(self, path):
             'properties': {},
         }
 
+    @property
+    def wellcome_client(self):
+        return StorageServiceClient(
+            api_url=self.api_root_url,
+            token_url=self.token_url,
+            client_id=self.app_client_id,
+            client_secret=self.app_client_secret,
+        )
+
     def delete_path(self, delete_path):
         LOGGER.debug('Deleting %s from Wellcome storage' % delete_path)
 
     def move_to_storage_service(self, src_path, dest_path, dest_space):
         """ Moves src_path to dest_space.staging_path/dest_path. """
-        LOGGER.debug('Fetching %s from %s (%s) on Wellcome storage' % (
+        LOGGER.debug('Fetching %s on Wellcome storage to %s (space %s)' % (
```

Comment:
nit:

```suggestion
        LOGGER.debug('Fetching %s on Wellcome storage to %s (space %s)', 
```

Drop the `%`-formatting, use the more efficient comma-formatting used elsewhere.